src/util.cc

```diff
@@ -2182,13 +2182,14 @@ void get_memory_usage(uint64_t* total, uint64_t* malloced) {
     uint64_t ret_total;
 
 #if defined(HAVE_MALLINFO2) || defined(HAVE_MALLINFO)
+    if ( malloced ) {
```

Comment:
@vpax I don't think this comment was resolved.

Reply:
I took Arne's comment to mean "in principle we should deprecate the parameter", not "this PR should deprecate it". It seems a bit funny to do it via this PR since the code is just a(nother) user of the API, isn't otherwise touching it, except to make a (major) performance improvement. But if you want to do the deprecation with this PR, I'm game to add it.

Reply:
That's a reasonable take. We can do it separately.